### ISSUE TITLE
rm's del command on Windows recurses beyond selection

### ISSUE THREAD

**Issue by AeliusSaionji** [Canop/broot#627]
I had an empty file named `2` hanging around my homedir. Used broot to remove `2` and `2` alone.
It started recursing into sub directories and deleted a bunch of files in my Appdata. 😨  ❗ 

```cmd
Deleted file - C:\Users\aelius\2
Deleted file - C:\Users\aelius\AppData\Local\EpicGamesLauncher\Saved\webcache_4147\IndexedDB\https_store-launcher.epicgames.com_0.indexeddb.blob\1\00\2
Deleted file - C:\Users\aelius\AppData\Local\Microsoft\Edge\User Data\Default\databases\https_www.ebay.com_0\2
Deleted file - C:\Users\aelius\AppData\Local\Microsoft\Edge\User Data\Default\IndexedDB\https_us02web.zoom.us_0.indexeddb.blob\2\00\2
Deleted file - C:\Users\aelius\AppData\Local\Microsoft\Edge\User Data\Default\IndexedDB\https_web.skype.com_0.indexeddb.blob\1\00\2
Deleted file - C:\Users\aelius\AppData\Local\Microsoft\Edge\User Data\Default\IndexedDB\https_www.amazon.com_0.indexeddb.blob\2\00\2
```

The problem is broot uses the /S switch on `del`. I didn't know about this behavior either, very surprised something like this exists.

From `del /?`

```
/S            Delete specified files from all subdirectories.
```

Demonstration
```
C:\Users\aelius\tmp>tree . /F
Folder PATH listing
Volume serial number is 00000049 ACE6:1E7E
C:\USERS\AELIUS\TMP
│   2
│
└───folder
        2

C:\Users\aelius\tmp>del /S 2
Deleted file - C:\Users\aelius\tmp\2
Deleted file - C:\Users\aelius\tmp\folder\2
```

Fix should be as simple as removing that `/S` flag from the del invocation that `:rm` brings up, and anywhere else it may be hiding.

**Comment by Canop** [comment]
Oo What ? This `/s` behavior looks like a joke.

@Spacelord-XaN Do you remember why you've put this flag ?

**Comment by Spacelord-XaN** [comment]
I'm sorry but I cannot remember why I put the /S flag there.
I can just point to my comment in the  PR (https://github.com/Canop/broot/pull/536#issue-1206340880):
'The flags that are passed to the respective windows commands are open do discussion as I am not sure if they behave the same like in unix.'

Regarding to the behavior that is shown here, I would remove that flag.
I don't think that anyone expects remove to behave like this.

**Comment by Canop** [comment]
> Regarding to the behavior that is shown here, I would remove that flag.

I'll do that and release a new version.

Before that, a new look by a windows user on the flags in [this file](https://github.com/Canop/broot/blob/master/src/verb/builtin.rs) would be a good idea.

copy file or dir:
    xcopy /Q /H /Y /I {file} {newpath:path-from-parent}

make directory:
    cmd /c mkdir {subpath:path-from-directory}

move file or dir:
    cmd /c move /Y {file} {newpath:path-from-parent}

rename file or dir:
    cmd /c move /Y {file} {parent}/{new_filename}

**Comment by Canop** [comment]
Doing the same for rmdir which was

        external("rm", "cmd /c rmdir /Q {file}", StayInBroot)
            .with_stype(SelectionType::Directory),

**Comment by Canop** [comment]
What about rmdir @AeliusSaionji ? It's defined as

        #[cfg(windows)]
        external("rm", "cmd /c rmdir /Q /S {file}", StayInBroot)
            .with_stype(SelectionType::Directory),

**Comment by AeliusSaionji** [comment]
Looks like that one should stay, it's equivalent to unix `rm -rf`

```
C:\Users\aelius>rmdir /?
Removes (deletes) a directory.

RMDIR [/S] [/Q] [drive:]path
RD [/S] [/Q] [drive:]path

    /S      Removes all directories and files in the specified directory
            in addition to the directory itself.  Used to remove a directory
            tree.

    /Q      Quiet mode, do not ask if ok to remove a directory tree with /S
```

**Comment by AeliusSaionji** [comment]
> Before that, a new look by a windows user on the flags in [this file](https://github.com/Canop/broot/blob/master/src/verb/builtin.rs) would be a good idea.

The rest of the cfg(windows) entries look good to me 👍

**Comment by Canop** [comment]
Released 1.16.2: https://github.com/Canop/broot/releases/tag/v1.16.2

**Comment by AeliusSaionji** [comment]
Tested working 👍
